src/collections/_documentation/platforms/python/index.md

```diff
@@ -291,8 +291,8 @@ The Python SDK provides some common [hints](/error-reporting/configuration/filte
 
 ## Troubleshooting and known issues
 
-This section contains various gotchas that are inherent to the Python SDK's
-design, do not necessarily affect the majority of users and do not really fit
-anywhere else.
+Sometimes a little advice from a friend is all it takes to solve a problem.
```

Comment:
I like the friendly tone, but perhaps we might simply say "The [Troubleshooting](/platforms/python/troubleshooting) section provides advice and tips to help solve common problems that may occur when using the Python SDK."

Reply:
That works for me too, except perhaps for the "common problems". At least for the current content the page is more about the "esoteric" side of problems, problems that only happen under uncommon circumstances / combinations of configurations.

Reply:
I made it short and direct:

```
The Troubleshooting page provides advice for solving known problems.
```